cascade-api/src/Cascade/Api/Database/Task.hs

```diff
@@ -0,0 +1,41 @@
+module Cascade.Api.Database.Task
```

Comment:
I'm changing the table modules name to `*Table`. So can we adopt the new naming (TaskTable) here too? If you don't mind about the changes. If you do, I can merge this branch into the #28 after this one merged into the `next` and I do the changes.

Reply:
It's fine for now. We will change it later.